docs/Configuration.md

```diff
@@ -27,6 +29,33 @@ MySQL             | `MySQL`
 PostgreSQL        | `PostgreSQL`
 SQLite            | `SQLite`
 
+### user
+
+The name of the user with sufficient permission to access the database.
+
+Ignored by the SQLite database engine.
+
+### password
+
+The password of the configured user.
+
+Ignored by the SQLite database engine.
+
+### database_host
+
+The host name of the machine on which the engine is running.
+
+Ignored by the SQLite database engine.
+
+### database_name
+
+The name of the database to use, except for SQLite database engine where
+it holds the full path to the SQLite database file.
+
+### polling_interval
+
+Time in seconds between database lookups by Test Agent.
```

Comment:
Just a side comment, the "polling_interval" key does not really belong to the "DB" section (but that is how it is).